website/docs/cli/commands/workspace/delete.mdx

```diff
@@ -13,20 +13,25 @@ Usage: `terraform workspace delete [OPTIONS] NAME [DIR]`
 
 This command will delete the specified workspace.
 
-To delete an workspace, it must already exist, it must have an empty state,
-and it must not be your current workspace. If the workspace state is not empty,
+To delete a workspace, it must already exist, it must not have resources under management,
+and it must not be your current workspace. If the workspace has resources under management,
 Terraform will not allow you to delete it unless the `-force` flag is specified.
 
-If you delete a workspace with a non-empty state (via `-force`), then resources
+Workspaces with resources under management (RUM) are workspaces which manage resources that physically exist.
```

Comment:
Should we introduce this acronym? Is it a term we want to familiar users with to the extend that we want to use it in the docs? It seems like we usually spell out words like Terraform Cloud instead of using acronyms like TFC

Reply:
good feedback! For a while there it seemed like we were going to make RUM a thing, but since we are not I have updated this to talk about tracked resources instead, as per your other comment